src/ssh/cockpitsshrelay.c

```diff
@@ -556,11 +556,11 @@ session_has_known_host_in_file (const gchar *file,
                                 const guint port)
 {
 #if !HAVE_DECL_SSH_SESSION_HAS_KNOWN_HOSTS_ENTRY
-  shim_set_knownhosts_file(file);
+  shim_set_knownhosts_file (file ? file : data->user_known_hosts);
```

Comment:
The tiniest nitpick, but `value ?: fallback` is a bit more succinct. (SCNR)

Reply:
Haha cool, I didn't even knew that syntax existed :D